### PR TITLE
A few bug fixes in the Thermal Scanner missions

### PR DESCRIPTION
**Pull request by TheGiraffe3**
I've now actually had time to test a bit more, and this is what came up.

### CODE REVIEW

File: data/governments.txt [hunk 1]
```diff
@@ -1260,7 +1260,9 @@ government "Kraz Cybernetics"
 	swizzle 2
 	color "governments: Free"
 	"player reputation" 0
-	"friendly hail" "friendly free worlds"
```
Comment (Zitchas) [inline data/governments.txt:1263]: Is this intentionally removing the friendly hails?

Comment (TheGiraffe3) [reply data/governments.txt:1263]: Probably should have discussed this a bit more in the opening post.
Does it make sense for the Kraz ship to hail like a FW ship? "We are the protectors of this system." but "we" doesn't work in the first place - and this ship hardly even has weapons.
But I don't think the merchant hails exactly work either... Barmy Edward is probably an employer who doesn't let his employees go three weeks away.
It's a complicated question, and I'd be interested in hearing your thoughts on it.

Comment (Zitchas) [reply data/governments.txt:1263]: Those are good points. There really should be some more corporate hails. Things along the lines of the regular friendly merchant ones, but adjusted to reflect coming from an employee in a medium company. 

I think in this case, probably sticking with the friendly merchant ones is better than not having any, but having a culled subset of those that fits better would be good.

File: data/governments.txt [hunk 2]
```diff
@@ -1260,7 +1260,9 @@ government "Kraz Cybernetics"
...
+	"attitude toward"
```
Comment (TheGiraffe3) [inline data/governments.txt:1263]:
```suggestion
	"friendly hail" "friendly Katz cybernetics"
	"attitude toward"
```

File: data/governments.txt (hunk 1 above)
Comment (TheGiraffe3) [reply data/governments.txt:1263]: Alright. Added a `friendly kraz cybernetics` hail set that should resolve this.

**Pull request merged**